ui/src/app/views/workspace/component.html

```diff
@@ -0,0 +1,61 @@
+<h2>AoU Research Portal Workspace</h2>
+<div *ngIf="user else loggedout">
+  <p class="description-text">Default workspace for July, 2017 demo</p>
+  <div class="area-block">
+    <h3>CDR Version</h3>
+    <div class="indented">
+      <!-- TODO: Add CDR Version Info - blrubenstein -->
+      <p>CDR version info goes here.</p>
+    </div>
+  </div>
+  <div class="area-block">
+    <h3>Cohorts</h3>
+    <!-- TODO: Add Cohort Link - blrubenstein -->
+    <a href="javascript:;">Add a Cohort</a>
+    <div class="indented">
+      <div>
+        <div class="table-header">
+          <div>Name</div>
+          <div class="wider-table">Description</div>
+          <div>URL</div>
+          <div>Created Date</div>
+        </div>
+        <div *ngFor="let cohort of cohortList" class="table-row">
+          <!-- TODO: Edit Cohort Link - blrubenstein -->
+          <div><a href="javascript:;">{{cohort.name}}</a></div>
+          <div class="wider-table">{{cohort.description}}</div>
+          <div>/cohort/{{cohort.id}}</div>
```

Comment:
This should be scoped now. I also realized my scoping for workspaces never went through, so that should also be in now.